### DISCUSSION TITLE
ENABLE_RESTRICTED_APP_ACCESS Setting

### DISCUSSION THREAD

**Discussion by ezrajrice** [tethysplatform/tethys#632] [Q&A] [answered]
How does the Tethys Setting ENABLE_RESTRICTED_APP_ACCESS work? I had it enabled and for a user, I gave them the permission to view the app, but still couldn't when they logged in.

**Comment by swainn** [comment, marked as answer]
@ezrajrice the app access permission needs to be assigned to a Group and then add the user to that group. The Change Group dialog now has an Apps section where you can add the appropriate app access permission to the group. This will only be enforced if ENABLE_RESTRICTED_APP_ACCESS is enabled. Assigning the permission directly to the users through the permissions dialog will not work at the present time. This is documented, but not very clearly: http://docs.tethysplatform.org/en/stable/tethys_portal/admin_pages.html#assign-app-permission-groups
